client/src/components/profile-box.js

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect } from 'react';
+import { Form, Icon, Input, Modal, Row, Col } from 'antd';
+import { AuthStore } from '../stores';
+import { inject } from '../utils';
+import 'draft-js-static-toolbar-plugin/lib/plugin.css';
+
+const ProfileBox = ({
+  stores: [authStore],
+  form: {
+    getFieldDecorator,
+    validateFields,
+    setFieldsValue,
+    getFieldValue,
+    getFieldsValue,
+  },
+  visible,
+  toggleModal,
+  initialProfile,
```

Comment:
You don't need this anymore since we directly inject 'AuthStore' into `ProfileBox` to retrieve the initial value